generative/maisi/scripts/find_masks.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import json
+import os
+import zipfile
+
+
+def convert_body_region(
+    body_region: list[int],
```

Comment:
How about using a dict to represent the map:

```
region_mapping = {
    "head": 0,
    "chest": 1,
    "thorax": 1,
    "chest/thorax": 1,
    "abdomen": 2,
    "pelvis": 3,
    "lower": 3,
    "pelvis/lower": 3,
}

body_region_indices = []
for region in body_region:
    normalized_region = region.lower()
    if normalized_region not in region_mapping:
        raise ValueError(f"Invalid region: {normalized_region}")
    body_region_indices.append(region_mapping[normalized_region])
  ```
  
  Also, there is no enforcement of function argument list should start a new line. Please feel free to do:
  ```
  def convert_body_region(body_region: List[str]) -> List[int]:
  ```

Reply:
Awesome! Thanks, will do